<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>图像显示</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>521</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>学生签到信息</string>
       </property>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>251</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>学生检测信息</string>
       </property>
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>251</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>197</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>开始签到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>停止签到</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>班级信息管理</string>
    </property>
    <addaction name="actionaddclass"/>
    <addaction name="actiondelclass"/>
    <addaction name="actionfindclass"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>学生信息管理</string>
    </property>
    <addaction name="actionaddStu"/>
    <addaction name="actiondelStu"/>
    <addaction name="actionfindStu"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionaddclass">
   <property name="text">
    <string>添加班级</string>
   </property>
  </action>
  <action name="actiondelclass">
   <property name="text">
    <string>删除班级</string>
   </property>
  </action>
  <action name="actionfindclass">
   <property name="text">
    <string>班级查找</string>
   </property>
  </action>
  <action name="actionaddStu">
   <property name="text">
    <string>增加学生</string>
   </property>
  </action>
  <action name="actiondelStu">
   <property name="text">
    <string>删除学生</string>
   </property>
  </action>
  <action name="actionfindStu">
   <property name="text">
    <string>学生查找</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
